### ISSUE TITLE
SMB check fails (expected SMB data check to be size 3 but was size 4

### ISSUE THREAD

**Issue by bbrala** [hashicorp/vagrant#10151]
2.1.3 breaks my SMB connection... The check for correct json fails.

### Vagrant version
2.1.3

### Host operating system
Windows 10 1804 (Dutch)

### Guest operating system
Ubuntu 16.04

### Debug output
https://gist.github.com/bbrala/775b53fe368ed4e2e164aba9e347a7dc

### Expected behavior
Box should start

### Actual behavior
Box hangs ar SMB check

### Steps to reproduce
1. vagrant up

**Comment by bbrala** [comment]
Ok, more issues, but perhaps related. When destroying a VM, it hands on deleting box. I also tried this at home (english windows 10 1804) same issue.

Vagrantfile:
```
# -*- mode: ruby -*-
# vi: set ft=ruby :


Vagrant.configure("2") do |config|
  config.vm.box = "generic/ubuntu1604"

  config.vm.provider "hyperv" do |v, override|
	override.vm.synced_folder "../", "/home/localcopy", smb_id: "projecten70hyperv", type: "smb", mount_options: ["mfsymlinks", "forceuid", "noperm", "file_mode=0777", "dir_mode=0777", "vers=3.0"]
  end
end
```

And the gist with the clean vagrant file:

https://gist.github.com/bbrala/261454af8d5368c281cf02a4ae52e461

**Comment by bitleader** [comment]
Same here.

**Comment by chrisroberts** [comment]
@bbrala Thanks for the report. There was a PR merged in the last release to help remove the dependence on key text which caused issues on non-english systems. However, depending on the share setup, there can be extra fields included. PR to fix this is #10156 and will be included in a patch release I'm prepping up today.

Thanks again!

**Comment by bbrala** [comment]
Thanks for the quick turn around, just had everyone update their setup, hopefully the slow ones wont end up on this broken release. :)
